src/ui/components/shared/UserSettingsModal/ExperimentalSettings.tsx

```diff
@@ -64,6 +64,11 @@ const EXPERIMENTAL_SETTINGS: ExperimentalSetting[] = [
     description: "Enable backend support for running processing routines (like React DevTools)",
     key: "enableRoutines",
   },
+  {
+    label: "Retry Backend Processing Routines",
```

Comment:
Oi, we are all over the place with the capitalization in these labels huh?

Reply:
hah, I can change that :)